helm/polaris/Chart.yaml

```diff
@@ -21,8 +21,8 @@ apiVersion: v2
 name: polaris
 description: A Helm chart for Apache Polaris (incubating)
 type: application
-version: 0.1.0
-appVersion: 1.0.0-incubating-SNAPSHOT
+version: 1.0.0
+appVersion: 1.0.0-incubating
```

Comment:
This is the docker image tag. The 1.0.0-incubating docker image will be published after the vote passed per release guide suggsted.

Reply:
`appVersion` has nothing to do with the docker image tag. The docker image tag is set in `values.yaml` at `image.tag`.

Reply:
Here is an example, which assumes Helm chart version matched with Polaris src/bin/docker version:
1. When an user revert to the Helm Chart v1.0.0, it still points the `latest` docker image, which could be 1.1.0 already. This introduces the incompatible risks. On the other hand, if we specified a version(e.g., 1.0.0) instead of `latest`, the Helm reversion will always be same.
2.  if people install the latest version of the helm chart, it still points to the latest docker image as we release them together. So that use case is covered as well.

With that, I think specifying a version is preferrable than `latest`. WDYT?

Reply:
That's a valid point! `latest` is not very friendly to historical chart versions. I am OK with using the release version instead.